### ISSUE TITLE
Roses are white but often red (SOLVED)

### ISSUE THREAD

**Issue by v0rl0x** [puzzlehunt/gsmgio-5btc-puzzle#20]
TO START OFF: It said to go back to the first puzzle.

RELATING TO THE PUZZLE and the keywords on phase 3.2 about the matrix:

last words before archichoice (architect character in the matrix)

matrix sum list or matrixsumlist

This relates to the matrix and how red and blue pills is a choice to get out of it. Now how does red and blue relate to the roses are white phrase?

When it stated Roses are White but often Red. Yellow has a number and so does Blue.

We replaced the white squares with red squares (or u can just imagine that white is red and so on and then calculate the binary) previously, but in the matrix you have red and blue pill what does this mean?

What if we give only red squares (white squares) and blue squares as the number 1 then yellow squares and black squares black as the number 0 for the binary translation.

Remember from the previous puzzle and how we were messing with the roses are white thing and using spiral, etc but this time red(white) and blue = 1.

When somebody did the puzzle originally with Yellow (0), Blue (1), White (0), Black (1) it led to the sequence as follows when doing spiral values, binary, etc:

0 0 1 1 0 1 0 0 1 0 1 1 0 0
1 1 1 1 0 0 1 1 1 0 1 0 1 1
1 1 0 1 1 1 0 1 0 0 1 0 0 1
0 1 1 0 1 0 0 0 0 1 1 1 0 1
0 1 1 0 0 0 1 1 0 0 0 1 1 0
1 0 0 1 1 0 0 0 1 0 0 0 1 1
1 0 0 1 1 1 0 0 0 1 0 0 0 0
1 1 1 0 0 0 0 0 0 0 1 0 0 0
0 0 0 1 1 1 0 1 1 1 1 1 0 1
1 1 1 1 1 1 0 0 1 1 0 0 0 1
1 1 0 1 0 0 0 0 0 1 1 0 1 1
1 1 1 1 0 0 1 0 1 0 1 1 0 0
0 1 0 1 1 1 0 1 0 0 0 1 1 0
0 1 1 0 1 1 0 1 1 0 1 0 1 1

Which converted to ASCII led to this URL which solved that phase:

01100111 (103 g) 01110011 (115 s) 01101101 (109 m) 01100111 (103 g) 00101110 (46 .) 01101001 (105 i) 01101111 (111 o) 00101111 (47 /) 01110100 (116 t) 01101000 (104 h) 01100101 (101 e) 01110011 (115 s) 01100101 (101 e) 01100101 (101 e) 01100100 (100 d) 01101001 (105 i) 01110011 (115 s) 01110000 (112 p) 01101100 (108 l) 01100001 (97 a) 01101110 (110 n) 01110100 (116 t) 01100101 (101 e) 01100100 (100 d)

And that led to:  gsmg.io/theseedisplanted

But when we do red=1 and blue=1 we get:

1 1 0 0 1 0 1 1 0 1 0 0 1 1
0 0 0 0 1 1 0 0 0 1 0 1 0 0
0 0 1 0 0 0 1 0 1 1 0 1 1 0
1 0 0 1 0 1 1 1 1 0 0 0 1 0
1 0 0 1 1 1 0 0 1 1 1 0 0 1
0 1 1 0 0 1 1 1 0 1 1 1 0 0
0 1 1 0 0 0 1 1 1 0 1 1 1 1
0 0 0 1 1 1 1 1 1 1 0 1 1 1
1 1 1 0 0 0 1 0 0 0 0 0 1 0
0 0 1 0 1 1 1 1 1 0 0 1 1 0
0 0 1 0 1 1 1 1 1 0 1 0 0 0
0 0 0 0 1 1 0 1 0 1 0 0 1 1
1 0 1 0 0 0 1 0 1 1 1 0 0 1
1 0 0 1 0 0 1 0 0 1 0 1 0 0

IN ASCII which is:

01100111 (103 g) 01110011 (115 s) 01101101 (109 m) 01100111 (103 g) 00101110 (46 .) 01101001 (105 i) 01101111 (111 o) 00101111 (47 /) 01110100 (116 t) 01101000 (104 h) 01100101 (101 e) 01110011 (115 s) 01100101 (101 e) 01100101 (101 e) 01100100 (100 d) 01101001 (105 i) 01110011 (115 s) 01110000 (112 p) 01101100 (108 l) 01100001 (97 a) 01101110 (110 n) 01110100 (116 t) 01100101 (101 e) 01100100 (100 d)

And in the ASCII code we can see the URL: gsmg.io/theseedisplanted which was the exact same as before with the different binary values.

So in conclusion this solved the riddle about the roses being often red rather than white.

Now what does  gsmg.io/theseedisplanted have to do with anything? I thought there was only one solution possible for this puzzle. But apparently not because something stated this: 

"It might have shown you only one door, beware that the rabbits nest may contain a whole lot more."

So we have to find more openings in the gsmg.io/theseedisplanted

**Comment by guy29278** [comment]
that didn't solve the riddle or you would of unlocked a key to solve the puzzle there's a way to make some of the white squares turn red and when you do you will notice an image appear somewhere on the main image ;)

**Comment by mohin0111974** [comment]
sorry for reopen. it says that SOLVED. I checked the bin, but something doesn't add up, it turns out trash. Roses are white but often red and it is not clear again

**Comment by guy29278** [comment]
If you use Steganography you will realise there is more to the main image than meets the eye, if you change to the last red colour plane and look closely a secret key will appear in 3 locations on the image, which is what that phase is talking about once you make the top right corner square go red the key will show itself... If this gets you on the right track and you're able to solve the puzzle don't forget about me.

12fdR7eLWYCLd767wXMkLQnS3eVdm9Zi22

**Comment by guy29278** [comment]
It talking about yellow has a number and so does blue is just referring to converting the coloured squares into binary, meaning you will need to also convert the hidden key into binary in order to use it with the main puzzle. Happy hunting.

**Comment by guy29278** [comment]
Heads up finding the hidden key isn't an automatic solve either unless your much better at this then I, have already figured out how to disseminate it amongst the main puzzle like neo had to do with the prime program given to him by the oracle, but stuck on what to do with the new 14x14 that adding the new 1's and 0's from the key made. 

Been at this point for a little over a year after getting stuck kind of forgot about the puzzle for awhile. The fact it spirals inwards and is rather long gives alot of area to make a mistake and put the wrong digit or skip one by accident took a few tries to get the new digits in the right spots... 

The key is also made up of dots and dashs that need to be converted into 1's and 0's before they can be placed onto the 14x14 main puzzle. 

Unrelated nitbit to the puzzle itself, the matrix movie (the puzzle) came out in the 90's, there used to be a cartoon back then called dot and dash (the key), feel like whoever created this puzzle knew that and maybe even did it on purpose.

**Comment by guy29278** [comment]
Also steganography is used to solve most of these puzzles without using it you can't actually solve them, biggest hint no one is talking about...

Steganography is the practice of concealing a message within another message or physical object, so the hidden message is not obvious to an observer. It differs from cryptography, which scrambles data, by hiding the very existence of a secret message itself. Steganography can be used to hide text, images, audio, video, or even network traffic inside of one or other.

**Comment by wat96** [comment]
@guy29278 What do you mean the keys popped out? I did this and I see nothing? I checked all the red layers
